extensions/database/tests/src/com/google/refine/extension/database/DBExtensionTests.java

```diff
@@ -43,7 +43,7 @@
     protected final String DEFAULT_MYSQL_HOST = "127.0.0.1";
     protected final String DEFAULT_MYSQL_PORT = "3306";
     protected final String DEFAULT_MYSQL_USER = "root";
-    protected final String DEFAULT_MYSQL_PASSWORD = "secret";
+    protected final String DEFAULT_MYSQL_PASSWORD = "as2188988";
```

Comment:
I don't think we should change this. You should be able to change the db password in the xml settings: `extensions/database/tests/conf/tests.xml` (and not commit that file).

Reply:
Thank you ! I will revise it a little bit

Reply:
> I don't think we should change this. You should be able to change the db password in the xml settings: `extensions/database/tests/conf/tests.xml` (and not commit that file).

By the way, during I write the test case, I find that change configuration in the `extensions/database/tests/conf/tests.xml` cannot finish the MySQL configuration correctly. I am not sure about that. I will do more test later